tests/integration/test_wazuh_db/test_db_backup.py

```diff
@@ -0,0 +1,216 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Wazuh-db is the daemon in charge of the databases with all the Wazuh persistent information, exposing a socket
+       to receive requests and provide information. The Wazuh core uses list-based databases to store information
+       related to agent keys, and FIM/Rootcheck event data.
+       This test checks the usage of the backup command used generate backups and restore the database using backups
+       generated with this same command.
+
+tier: 0
+
+modules:
+    - wazuh_db
+
+components:
+    - manager
+
+daemons:
+    - wazuh-db
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-db.html
+
+tags:
+    - wazuh_db
+'''
+from pathlib import Path
+import time
+import pytest
+
+from wazuh_testing.constants.paths import WAZUH_PATH
+from wazuh_testing.constants.paths.sockets import WAZUH_DB_SOCKET_PATH
+from wazuh_testing.constants.executions import TIER0, SERVER, LINUX
+from wazuh_testing.utils.database import query_wdb
+from wazuh_testing.utils import configuration
+
+from . import TEST_CASES_FOLDER_PATH
+
+
+# Marks
+pytestmark = [TIER0, LINUX, SERVER]
+
+# Configurations
+t_cases_path = Path(TEST_CASES_FOLDER_PATH, 'cases_wazuh_db_backup_command.yaml')
+t_config_parameters, t_config_metadata, t_case_ids = configuration.get_test_cases_data(t_cases_path)
+
+backups_path = Path(WAZUH_PATH, 'backup', 'db')
+receiver_sockets_params = [(WAZUH_DB_SOCKET_PATH, 'AF_UNIX', 'TCP')]
+monitored_sockets_params = [('wazuh-db', None, True)]
```

Comment:
Use the `'wazuh-db'` and name defined in the framework (`WAZUH_DB_DAEMON`). Apply this change in all the places that you use it:
```suggestion
monitored_sockets_params = [(WAZUH_DB_DAEMON, None, True)]
```